src/System.Management.Automation/engine/SessionStateDriveAPIs.cs

```diff
@@ -508,13 +508,13 @@ private PSDriveInfo GetDrive(string name, bool automount)
 
             if (result == null && automount)
             {
-                // first try to automount as a file system drive
-                result = AutomountFileSystemDrive(name);
-                // if it didn't work, then try automounting as a BuiltIn drive (e.g. "Cert"/"Certificate"/"WSMan")
-                if (result == null)
-                {
-                    result = AutomountBuiltInDrive(name); // internally this calls GetDrive(name, false)
-                }
+                // first try to automount as a file system drive
+                result = AutomountFileSystemDrive(name);
+                // if it didn't work, then try automounting as a BuiltIn drive (e.g. "Cert"/"Certificate"/"WSMan")
+                if (result == null)
+                {
+                    result = AutomountBuiltInDrive(name); // internally this calls GetDrive(name, false)
+                }
```

Comment:
I wonder what is changed here? EOL?

Reply:
Git gave some warnings about line endings (and performing some LF<->CRLF conversions on my behalf since I was doing this on Windows), so I think that's what this is.

Reply:
Thanks for clarify. 
Closed.

Reply:
The original file had inconsistent line endings, `LF` everywhere but this region. Your script changed the file to have consistent line endings, so git sees this region differently I guess.